### ISSUE TITLE
 BlitCommandEncoder.CopyFromTexture doesn't work

### ISSUE THREAD

**Issue by TotallyGamerJet** [ebitengine/purego#290]
### PureGo Version

main

### Operating System

- [ ] Windows
- [X] macOS
- [ ] Linux
- [ ] FreeBSD
- [ ] Android
- [ ] iOS

### Go Version (`go version`)

go version go1.22.1 darwin/arm64

### What steps will reproduce the problem?

See https://github.com/hajimehoshi/ebiten/issues/3135

### What is the expected result?

Either purego should properly panic that there are too many arguments or it should work as expected

### What happens instead?

It crashes with an OS error:

```
2024-10-19 23:20:43.969 2048[80611:175264348] [CAMetalLayer nextDrawable] returning nil because allocation failed.
-[IOGPUMetalCommandBuffer validate]:216: failed assertion `commit command buffer with uncommitted encoder'
SIGABRT: abort
PC=0x7ff8058c714a m=4 sigcode=0
signal arrived during cgo execution
```

### Anything else you feel useful to add?

I think it's better to error then to add support for more arguments

**Comment by hajimehoshi** [comment]
> I think it's better to error then to add support for more arguments

I agree. Let's have a better error message, and revisit this whenever we want :)
